src/m365/planner/commands/bucket/bucket-get.ts

```diff
@@ -0,0 +1,215 @@
+import { PlannerBucket } from '@microsoft/microsoft-graph-types';
+import { Logger } from '../../../../cli';
+import { AxiosRequestConfig } from 'axios';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  title?: string;
+  planId?: string;
+  planName?: string;
+  ownerGroupId?: string;
+  ownerGroupName?: string;
+}
+
+class PlannerBucketGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.BUCKET_GET;
+  }
+
+  public get description(): string {
+    return 'Gets the Microsoft Planner bucket in a plan';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.title = typeof args.options.title !== 'undefined';
+    telemetryProps.planId = typeof args.options.planId !== 'undefined';
+    telemetryProps.planName = typeof args.options.planName !== 'undefined';
+    telemetryProps.ownerGroupId = typeof args.options.ownerGroupId !== 'undefined';
+    telemetryProps.ownerGroupName = typeof args.options.ownerGroupName !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'name', 'planId', 'orderHint'];
+  }
```

Comment:
This operation is not being used in this command. You typically see it when you are executing a list command